hw_12.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+
+declare(strict_types=1);
+
+error_reporting(E_ALL);
+ini_set('display_errors', '1');
+
+//Task 1
+//
+//function writeMsg(): string
+//{
+//    return "Hello, World!";
+//}
+//
+//echo writeMsg();
+
+
+//Task 2
+//function leapYear(int $year): bool
+//{
+//    if (($year > 0 && $year % 4 === 0 && $year % 100 != 0) || ($year % 400 === 0 && $year % 100 === 0)) {
+//        return true;
+//    } else {
+//        return false;
+//    }

```

Comment:
операция в блоке if сразу возвращает bool
поэтому можно так
```suggestion
//    return ($year > 0 && $year % 4 === 0 && $year % 100 != 0) || ($year % 400 === 0 && $year % 100 === 0);
```